program_repair.py

```diff
@@ -0,0 +1,3 @@
+def repair(input_string):
```

Comment:
I think it is better if we rename this function into remove_leading_spaces so it is clearer what it does and that we will have other repair methods too.

Reply:
Done! I was planning on changing the name once I made another one.